common/docsV1.md

```diff
@@ -249,6 +249,22 @@ Public keys verify that client token requests are coming from an authorized appl
 
 **1. Download the snippet.txt file located in the DPC Portal to create a signature.**
 
+- You are also able to download the snippet.txt file using the command invocation below:
```

Comment:
We need some kind of call out in the docs stating "if you see this error when uploading your public keys, please refer to these instructions for downloading the snippet.txt and re-generate your public and private keys..." or something of the like